server/app/routes/lecture.js

```diff
@@ -50,16 +60,43 @@ router.post('/', function(req, res, next) {
 
 router.put('/:lectureId', function(req, res, next) {
   var io = req.app.get('socketio');
+
   req.lecture.update(req.body)
   .then(function(updatedLecture) {
     if (req.body.startTime && !req.body.endTime) {
       io.emit('startLecture', updatedLecture);
```

Comment:
For a future issue/PR maybe consider using socket.io [rooms](http://socket.io/docs/rooms-and-namespaces/). We could even use the lecture id as the room identifier. That way only people connected to that lecture will receive these messages.